odata-csdl/12 Action and Function.md

```diff
@@ -186,6 +186,9 @@ entities MAY specify an entity set path. The entity set path specifies the canon
 (as defined in [#OData-Protocol#ContextURL]) of the
 returned entities in terms of the binding parameter value.
 
+If no entity set path is specified or the canonical collection cannot be determined
+by the following rules, the service SHOULD include a context URL [#OData-Protocol#ContextURL] in the response.
+
```

Comment:
Delete this paragraph and instead rewrite [OData-Protocol, sections 10.2 and 10.3] where the entity set path is mentioned.